fisher.fish

```diff
@@ -181,7 +187,7 @@ end
 function _fisher_commit -a cmd
     set -e argv[1]
     set -l elapsed (_fisher_now)
-    set -l fishfile $fisher_path/fishfile
+    set -l fishfile $fish_config/fishfile
```

Comment:
What's up with this? Didn't we define `$fishfile` earlier in line 12?

Reply:
@Scrumplex - The `-l` flag defines a local variable in the scope of the function, so it needs to be done in both places if we want to use locals. I could convert the one on line 12 in the main `fisher` function to global (`-g`). Stylistically, it's probably fine in this project to do it that way - not my typical choice to use globals this way, but if it causes heartburn to define it in two places and a global is preferred, we can do that. @jorgebucaran - do you have a preference here?

Reply:
Going to agree with @Scrumplex, but only if `fishfile` can't change. See https://github.com/jorgebucaran/fisher/pull/547/files#r339362195.